src/darwin/Framework/CHIP/MTRNOCChainIssuer.h

```diff
@@ -22,6 +22,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef BOOL (^MTRNOCChainGenerationCompleteHandler)(NSData * operationalCertificate, NSData * intermediateCertificate,
```

Comment:
That BOOL should not be there.  It's there in some other functions right now, but in the process of being removed...

Reply:
Done. Returning nothing/void instead.